rosplan_stage_demo/config/costmap_common_params_waffle.yaml

```diff
@@ -1,10 +1,10 @@
 obstacle_range: 3.0
 raytrace_range: 3.5
 
-footprint: [[-0.06, -0.06], [-0.06, 0.06], [0.06, 0.06], [0.06, -0.06]]
-robot_radius: 0.30
+footprint: [[-0.205, -0.155], [-0.205, 0.155], [0.077, 0.155], [0.077, -0.155]]
```

Comment:
Was not sure why this was changed, but this are the original parameters for the waffle so I put them back.